### ISSUE TITLE
High CPU usage

### ISSUE THREAD

**Issue by vexxx1** [surfbryce/beautiful-lyrics#185]
Hi
When im using your extension my cpu usage on spotify is around 20-40%
without your extension, its 1-5%
The problem also occurs with hardware acceleration OFF

**Comment by Marmur2020** [comment]
@vexxx1 I do not recommend disabling hardware acceleration, as it removes the load on the video card and loads more heavily on the processor and there may be subsequences: lags, freezes, and reduce the performance of the application.

**Comment by code-IM-perfect** [comment]
I can confirm this, I thought it was because of all the moving gradients in the lyrics and now playing page but this higher CPU usage also occurs when those panels are closed

**Comment by vexxx1** [comment]
> @vexxx1 I do not recommend disabling hardware acceleration, as it removes the load on the video card and loads more heavily on the processor and there may be subsequences: lags, freezes, and reduce the performance of the application.

Alr I enabled hardware acceleration and it seems like the CPU load decreased by around 7%. It still doesnt fix my problem tho, since the load is still above 20%

**Comment by DankestFke** [comment]
could we have an option to disable moving cover background and background blur (to stop the lag or just preference) to be honest, a dark accent background and a custom font could already work flawlessly for this extension

**Comment by Marmur2020** [comment]
Yeah, it's a good idea to make background modes like static, dynamic, or one color (like Spotify lyrics). Yeah, it's a good idea to make background modes like static, dynamic, or one color (like Spotify lyrics). I understand that the blurred background is not cached, but updating the blur effect every time also affects performance. To be honest, I noticed that it causes a lot of lag because of this.

**Comment by derfurkan** [comment]
Any updates on this? My CPU is being fried like a chicken

**Comment by RobyOneKenoby112** [comment]
Yeah, i'm still waiting, maybe some day
